packages/boa/tests/numpy/routines.matrix.js

```diff
@@ -9,48 +9,42 @@ boa.import('numpy.matlib');
 test('numpy.mat: interpret the input as a matrix.', t => {
   const x = np.array([[1, 2], [3, 4]]);
   const m = np.asmatrix(x);
-  t.equal(JSON.stringify(m.tolist()), '[[1,2],[3,4]]');
-  t.equal(type(m).__name__, 'matrix');
-  t.end();
+  t.is(JSON.stringify(m.tolist()), '[[1,2],[3,4]]');
```

Comment:
If `m.tolist()` return an array, we can use `deepEqual` like:
```js
t.deepEqual(m.tolist(), [[1,2],[3,4]]);
```

Reply:
tolist() won't return a normal JS array actually, that's why we used the string comparison here. And this PR is to migrate the framework only :(